data/mods/CRT_EXPANSION/scenarios/crt_classes.json

```diff
@@ -54,11 +54,11 @@
           "socks",
           "crt_dress_shoes",
           "id_military",
+		  "knife_swissarmy",
           "cell_phone"
         ],
         "entries": [
-          { "item": "water_clean", "ammo-item": "water_clean", "container-item": "thermos" },
-          { "item": "knife_swissarmy", "container-item": "crt_belt" }
+          { "item": "water_clean", "ammo-item": "water_clean", "container-item": "thermos" }
```

Comment:
This is the same line as line 60, it only marks line 60 as deleted because I had to remove the separator to make the array work.

Reply:
This is why trailing commas it is good practice, when programming in js. It's a shame JSON doesn't support them.